Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -82,25 +125,25 @@ class SettingsController: UITableViewController  {
         self.checkStartupSecurity()
 
         if Settings.networkMode == 0 {
-            network_mode_subtitle?.text = "Simplified Payment Verification"
+            network_mode_subtitle?.text = LocalizedStrings.spv
             self.certificate_cell.isUserInteractionEnabled = false
             self.server_cell.isUserInteractionEnabled = false
             self.connectPeer_cell.isUserInteractionEnabled = true
             self.server_ip.textColor = UIColor.lightGray
-            self.certificat_label.textColor = UIColor.lightGray
+            self.certificatDesc.textColor = UIColor.lightGray
             self.connect_peer_ip.textColor = UIColor.darkText
-            self.serverAdd_label.textColor = UIColor.lightGray
-            self.connect_ip_label.textColor = UIColor.darkText
+            self.serverAddDesc.textColor = UIColor.lightGray
+            self.connectIpDesc.textColor = UIColor.darkText
```

Comment:
What is `connectIp`? I'm sure you can use a more informative variable name. Also, `rpcServerAddressLabel` instead of `serverAddDesc`. Proper/informative/descriptive variable names make a codebase significantly easier to read and maintain.